apps/user/models/user.go

```diff
@@ -18,7 +18,8 @@ type User struct {
 	Email              string `gorm:"type:varchar(255); not null; unique_index"`
 	FirstName          string `gorm:"type:varchar(30); not null"`
 	LastName           string `gorm:"type:varchar(150)"`
-	Active             bool   `gorm:"default:true; not null"`
+	Password           []byte
```

Comment:
What would be the corresponding type in the PSQL database? Shouldn't we add a GORM tag for the same?

Reply:
done.